README.md

```diff
@@ -63,7 +63,7 @@ the form of an HTML table.
 You can use the `go` tool to install `grpcui`:
 ```shell
 go get github.com/fullstorydev/grpcui/...
```

Comment:
With older version of Go, you had to `go get` the package before installing it.

With the newer version, not only do you have to specify the version (if you are not in a module workspace), you also do not need to first do `go get`.

```suggestion
```